print-rsvp.c

```diff
@@ -1176,6 +1186,24 @@ rsvp_obj_print(netdissect_options *ndo,
             }
             break;
 
+        case RSVP_OBJ_CAPABILITY:
+            switch(rsvp_obj_ctype) {
+            case RSVP_CTYPE_1:
+                if (obj_tlen < 4)
+                    return-1;
+                ND_PRINT("%s  Flags: [%s]",
+                       indent,
+                       bittok2str(rsvp_obj_capability_flag_values,
+                                  "none",
+                                  GET_BE_U_4(obj_tptr)));
```

Comment:
"Reserved bits MUST be set to zero on transmission and MUST be ignored on receipt." (rfc5063)
Thus a mask is needed.

Reply:
Protocol speakers must ignore reserved bits.  Protocol *dumpers* might want to show them for 2 reasons
1. to say "Hey this sender is misbehaving, it is setting reserved bits to nonzero"
2. when a new feature comes out it can be debugged without having to change the mask.
That being said, `bittok2str` doesn't output anything for bits that are set that are not in the input values array, so, I'm still not quite sure what the issue is.

Reply:
> Protocol _dumpers_ might want to show them

Ok, proposed update
```patch
@@ -1193,11 +1193,15 @@ rsvp_obj_print(netdissect_options *ndo,
             case RSVP_CTYPE_1:
                 if (obj_tlen < 4)
                     return-1;
+                uint8_t unused_and_flags = GET_BE_U_4(obj_tptr);
+                if (unused_and_flags & ~RSVP_OBJ_CAPABILITY_FLAGS_MASK)
+                    ND_PRINT("%s  [reserved=0x%x must be zero]", indent,
+                        unused_and_flags & ~RSVP_OBJ_CAPABILITY_FLAGS_MASK);
                 ND_PRINT("%s  Flags: [%s]",
                        indent,
                        bittok2str(rsvp_obj_capability_flag_values,
                                   "none",
-                                  (GET_BE_U_4(obj_tptr) & RSVP_OBJ_CAPABILITY_FLAGS_MASK)));
+                                  (unused_and_flags & RSVP_OBJ_CAPABILITY_FLAGS_MASK)));
                 obj_tlen-=4;
                 obj_tptr+=4;
                 break;
```

Reply:
Just pushed the proposed changes. Thanks